lib/promisesdk.js

```diff
@@ -12,330 +12,342 @@ function ApigeeTool(defaults) {
 
 module.exports = ApigeeTool;
 
-ApigeeTool.defaults = function(newDefaults) {
-  var tool= new ApigeeTool(newDefaults);
+ApigeeTool.defaults = function (newDefaults) {
```

Comment:
Please undo the whitespace changes throughout this file in order to achieve a leaner and cleaner commit/diff.